app/views/tweets/index.erb

```diff
@@ -0,0 +1,41 @@
+<html lang="en">
```

Comment:
@portatlas, this is just a template and not the whole HTML page. As such, you don't need html tags. Assume you are already inside the <body> tag.

The whole layout is views/layout.erb. Each view page will go wherever `<% yield %>` is. 

@ftBessmann @buckmelton @Kristallam